src/test/java/com/lpvs/entity/config/WebhookConfigTest.java

```diff
@@ -1,3 +1,10 @@
+/**
+ * Copyright (c) 2022, Samsung Research. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2022, Samsung Electronics Co., Ltd. All rights reserved.
```

Reply:
Thanks for the comment. I've updated all headers.